docs/adr/adr-007-data-retention.md

```diff
@@ -0,0 +1,196 @@
+# Decision Record: Data Retention
+
+## Context
+
+The original ADR concerning data retention resides [here][initial policy] and
```

Comment:
I imagine it's worth adding a note in that ADR to say something like `This decision was revisited in [ADR-7](link) which supersedes this decision.`. I don't think you need to rename that file.

Reply:
Okay dokay, I've removed that commit and added a note in the previous ADR.